drivers/disk/sdmmc_stm32.c

```diff
@@ -372,6 +374,9 @@ static int stm32_sdmmc_access_init(struct disk_info *disk)
 	HAL_StatusTypeDef hal_ret;
 	int err;
 
+#ifdef CONFIG_PM_DEVICE_RUNTIME
+	pm_device_runtime_get(dev);
+#endif
```

Comment:
`pm_device_runtime_*()` are already stubbed when `CONFIG_PM_DEVICE_RUNTIME` is disabled.
```suggestion
	pm_device_runtime_get(dev);
```
Ditto below.